src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -400,6 +424,28 @@ public override bool Execute()
         }
     }
 
+    private bool ProcessNettrace(string netTraceFile)
+    {
+        var OutputMibcPath = Path.Combine(OutputDir, Path.ChangeExtension(Path.GetFileName(netTraceFile), ".mibc"));
+        (int exitCode, string output) = Utils.TryRunProcess(Log,
+                                                            PgoBinaryPath!,
+                                                            $"create-mibc --trace {netTraceFile} --output {OutputMibcPath}",
+                                                            silent: true,
+                                                            debugMessageImportance: MessageImportance.Low);
+
+        if (exitCode != 0)
+        {
+            Log.LogError($"dotnet-pgo failed for {netTraceFile} {output}");
+            return false;
+        }
+
+        MibcProfilePath = MibcProfilePath!.Concat(new string[]{OutputMibcPath}).ToArray();
+        Log.LogMessage(MessageImportance.High, $"MibcProfilePath is extended: \n{string.Join("\n", MibcProfilePath)}");
+
+        Log.LogMessage(MessageImportance.High, output);
```

Comment:
Does this need to be emitted `High`, which would always show in the build output for the user?
You can follow the pattern in https://github.com/dotnet/runtime/blob/main/src/tasks/AotCompilerTask/MonoAOTCompiler.cs#L833-L894 for the whole method.

Reply:
Ah, I wanted to see the message when testing this locally but forgot to change it to Low. Thanks!

Reply:
@radical I think a message that states the .mibc file was generated will be useful to show in the build output since that will be telling that a `.nettrace` was successfully consumed. Would this fit the criteria used for High importance? I'm not sure what criteria we are using for High vs Low.